docs/protocol/non-evm-implementations.mdx

```diff
@@ -0,0 +1,23 @@
+While our public documentation currently focuses on EVM compatibility, we offer varying levels of support for additional ecosystems. These ecosystems may not have full feature parity or complete tooling integration (such as TypeScript CLI support) at this time. We're actively working to expand our documentation and tooling across these platforms. If you're excited about building on top of these or bringing Hyperlane to other new ecosystems reach out on [our Discord](https://discord.gg/hyperlane)!
+
+# CosmWasm
+
+See the CosmWasm implementation of Hyperlane [here](https://github.com/many-things/cw-hyperlane), and follow [this guide](https://github.com/many-things/cw-hyperlane/blob/main/DEPLOYMENT.md) on how to deploy to a Cosmos appchain here. We're live on Celestia, Stride and Neutron as of July 2024.
```

Comment:
wonder if we should give explicit credit to Mitosis as we do for the other implementations?

Reply:
I think we should